lib/l10n/app_es.arb

```diff
@@ -0,0 +1,539 @@
+
+  "removeEntryMetadataDialogTitle": "Eliminación de metadatos",
+  "removeEntryMetadataDialogMore": "Más",
+
+  "removeEntryMetadataMotionPhotoXmpWarningDialogMessage": "XMP es necesario para reproducir la animación de una foto en movimiento.\n\n¿Está seguro de que desea removerlo?",
+
+  "videoSpeedDialogLabel": "Velocidad de reproducción",
+
+  "videoStreamSelectionDialogVideo": "Video",
+  "videoStreamSelectionDialogAudio": "Audio",
+  "videoStreamSelectionDialogText": "Subtítulos",
+  "videoStreamSelectionDialogOff": "Desactivado",
+  "videoStreamSelectionDialogTrack": "Pista",
+  "videoStreamSelectionDialogNoSelection": "No hay otras pistas.",
+
+  "genericSuccessFeedback": "¡Completado!",
+  "genericFailureFeedback": "Falló",
+
+  "menuActionConfigureView": "Ver",
+  "menuActionSelect": "Seleccionar",
+  "menuActionSelectAll": "Seleccionar todo",
+  "menuActionSelectNone": "Deseleccionar",
+  "menuActionMap": "Mapa",
+  "menuActionStats": "Estadísticas",
+
+  "viewDialogTabSort": "Ordenar",
+  "viewDialogTabGroup": "Grupo",
+  "viewDialogTabLayout": "Disposición",
+
+  "tileLayoutGrid": "Cuadrícula",
+  "tileLayoutList": "Lista",
+
+  "aboutPageTitle": "Acerca de",
+  "aboutLinkSources": "Fuentes",
+  "aboutLinkLicense": "Licencia",
+  "aboutLinkPolicy": "Política de privacidad",
+
+  "aboutUpdate": "Nueva versión disponible",
+  "aboutUpdateLinks1": "Una nueva versión de Aves se encuentra disponible en",
+  "aboutUpdateLinks2": "y",
+  "aboutUpdateLinks3": ".",
+  "aboutUpdateGitHub": "GitHub",
+  "aboutUpdateGooglePlay": "Google Play",
+
+  "aboutBug": "Reporte de errores",
+  "aboutBugSaveLogInstruction": "Guardar registros de la aplicación a un archivo",
+  "aboutBugSaveLogButton": "Guardar",
+  "aboutBugCopyInfoInstruction": "Copiar información del sistema",
+  "aboutBugCopyInfoButton": "Copiar",
+  "aboutBugReportInstruction": "Reportar en GitHub con los registros y la información del sistema",
+  "aboutBugReportButton": "Reportar",
+
+  "aboutCredits": "Créditos",
+  "aboutCreditsWorldAtlas1": "Esta aplicación usa un archivo TopoJSON de",
+  "aboutCreditsWorldAtlas2": "bajo licencia ISC.",
+  "aboutCreditsTranslators": "Traductores:",
+  "aboutCreditsTranslatorLine": "{language}: {names}",
+  
+  "aboutLicenses": "Licencias de código abierto",
+  "aboutLicensesBanner": "Esta aplicación usa los siguientes paquetes y librerías de código abierto.",
+  "aboutLicensesAndroidLibraries": "Librerías de Android",
+  "aboutLicensesFlutterPlugins": "Añadidos de Flutter",
+  "aboutLicensesFlutterPackages": "Paquetes de Flutter",
+  "aboutLicensesDartPackages": "Paquetes de Dart",
+  "aboutLicensesShowAllButtonLabel": "Mostrar todas las licencias",
+
+  "policyPageTitle": "Política de privacidad",
+
+  "collectionPageTitle": "Colección",
+  "collectionPickPageTitle": "Elegir",
+  "collectionSelectionPageTitle": "{count, plural, =0{Seleccionar} =1{1 elemento} other{{count} elementos}}",
+
+  "collectionActionShowTitleSearch": "Mostrar títulos",
```

Comment:
`collectionActionShowTitleSearch` and `collectionActionHideTitleSearch` below is for the feature to "show/hide the title filter", not "show/hide titles".